src/react/components/ButtonGroup.scss

```diff
@@ -1,51 +1,62 @@
 @import '../stylesheets/variables';
 
 .ButtonGroup {
+  font-weight: bold;
   display: grid;
   grid-template-columns: 1fr 1fr 1fr;
   cursor: pointer;
 
   .GroupedButton {
-    font-size: 12px;
+    font-size: 13px;
+    color: $color-text-alt;
     line-height: 14px;
-    padding-top: 8px;
-    padding-bottom: 8px;
+    padding: 8px 10px;
     text-align: center;
+
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
```

Comment:
@developish is the extra whitespace necessary?

Reply:
No